odk1-src/collect-connect-google.rst

```diff
@@ -38,3 +38,5 @@ Connecting to a Google Drive Account
    You have the option to specify a :guilabel:`Fallback submisison URL`. This is the URL of a Google sheet to which form submissions will be posted if the submitted form does not specify its own URL.
 
    If your forms will specify a submission URL, you can leave this setting empty. Otherwise, enter the URL of a Google sheet you would like to use.    
+
```

Comment:
Looking pretty good!

Let's make this a warning before the first step since it might result in problems if they don't know. And give people a hint what the warning is to save them a click. "Spreadsheets have `cell limits (currently 5 million)` above which..."

Reply:
Done.